### PR TITLE
add API to get cache stats

### PR DESCRIPTION
**Pull request by AlvaroVega**
Based on node-cache stats API

API to get info about node cache stats:
https://github.com/node-cache/node-cache#statistics-stats

curl -i -X GET http://localhost:11211/admin/cacheStats
HTTP/1.1 200 OK
X-Powered-By: Express
fiware-correlator: 6386f32a-3f5f-4f57-8b05-813ce3ed6607
Content-Type: application/json; charset=utf-8
Content-Length: 265
ETag: W/"109-JPQLnurOZTITjopfP0LhKQSKm7A"
Date: Wed, 18 May 2022 11:17:41 GMT
Connection: keep-alive
Keep-Alive: timeout=5

{"cacheSats":{"subservice":{"hits":0,"misses":0,"keys":0,"ksize":0,"vsize":0},"roles":{"hits":9,"misses":1,"keys":1,"ksize":34,"vsize":200},"user":{"hits":9,"misses":2,"keys":1,"ksize":183,"vsize":240},"validation":{"hits":9,"misses":1,"keys":1,"ksize":197,"vsize":6}}}

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add: new API to retrieve cache stats
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: new API to retrieve cache stats (GET /admin/cacheStats)
```

File: operations.md [hunk 2]
```diff
@@ -25,6 +26,22 @@ curl -X GET "http://localhost:11211/admin/log"
...
+PEP keeps a memory cache with some access about roles, domains, users and subservices. Related with this info is possible
+get an statistics about these caches using the following API:
+
+```
+curl -X GET "http://localhost:11211/admin/cacheStats"
+```
+Getting
+
+`json
+{"cacheStats":{"subservice":{"hits":0,"misses":0,"keys":0,"ksize":0,"vsize":0},"roles":{"hits":9,"misses":1,"keys":0,"ksize":0,"vsize":0},"user":{"hits":9,"misses":2,"keys":1,"ksize":183,"vsize":240},"validation":{"hits":9,"misses":1,"keys":0,"ksize":0,"vsize":0}}}
```
Comment (fgalan) [inline operations.md:40]: Rendering look a bit ugly...

imagen

I'd suggest to use ` ```json` (triple quote) and pretty print

Comment (AlvaroVega) [reply operations.md]: done in https://github.com/telefonicaid/fiware-pep-steelskin/pull/509/commits/d2955757ebc45515e0344e0868640d703a4799ad

File: operations.md [hunk 3]
```diff
@@ -25,6 +26,20 @@ curl -X GET "http://localhost:11211/admin/log"
 
 Every error message is identified with a prefix code in brackets. The code convention can be found in Apendix A.
 
+## <a name="Cache Stats"/> Cache Stats
+
+PEP keeps a memory cache with some access about roles, domains, users and subservices. Related with this info is possible
+get an statistics about these caches using the following API:
+
+```
+curl -X GET "http://localhost:11211/admin/cacheStats"
```
Comment (fgalan) [inline operations.md:35]: Would it make sense a method to reset cache metrics? Eg:

```
DELETE /admin/cacheStats
```

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
...but tests not yet passing

File: operations.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
-# Operations Manual: logs and alarms
+# Operations Manual: logs, alarms and stats
 ## Index
 
 * [Logs](#logs)
+* [Cache Stats](#stats)
```
Comment (fgalan) [inline operations.md:5]: With the new operation for reseting the cache at all, maybe it's better to name it this way:

```suggestion
* [Cache management](#cache)
```

File: operations.md [hunk 5]
```diff
@@ -25,6 +26,33 @@ curl -X GET "http://localhost:11211/admin/log"
 
 Every error message is identified with a prefix code in brackets. The code convention can be found in Apendix A.
 
+## <a name="Cache Stats"/> Cache Stats
```
Comment (fgalan) [inline operations.md:29]:
```suggestion
## <a name="cache"/> Cache management
```

Related with previous comment

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**